corehq/apps/case_search/utils.py

```diff
@@ -116,9 +117,15 @@ def _add_case_property_queries(self):
             )
             for removal_regex in remove_char_regexs:
                 to_remove = re.escape(removal_regex.regex)
-                value = re.sub(to_remove, '', value)
+                if isinstance(value, list):
+                    new_value = []
+                    for val in value:
+                        new_value.append(re.sub(to_remove, '', value))
+                else:
+                    value = re.sub(to_remove, '', value)
 
             if '/' in key:
+                assert not isinstance(value, list), "Multiple xpath queries for a query param is not supported"
```

Comment:
This should probably raise a `CaseFilterError` so that it get's handled in the view

Reply:
Good point, thanks! Addressed